### DISCUSSION TITLE
HA - hardware monitoraggio energetico

### DISCUSSION THREAD

**Discussion by mimelaine** [indomus/forum#3322] [HUB: Home Assistant]
Buonasera a tutti,

valutando come integrare il monitoraggio energetico (carico, consumo ed eventualmente poter disattivare carichi oltre soglia dei singoli elettrodomestici) sto trovando diverse difficoltà nella scelta dell'hardware.

Dove possibile metterò delle **Shelly Plus Plug S**, e sul quadro sono orientato sullo **Shelly Pro EM 50A** per il carico/consumo totale.
Ma ad esempio per l'induzione non ho presa (quindi posso monitorarlo con la seconda pinza amperometrica dell'EM, ma non spegnerla), per il condizionatore nemmeno (è già smart di suo, quindi posso spegnerlo, ma per monitorare il consumo avrei bisogno di un altro EM, avendo già occupato entrambe le pinze), senza contare che spostare forno e lavastoviglie per mettere una Plug S non è banale, e fra l'altro non sono nemmeno certo che siano adatte (secondo voi il forno lo regge?)

Detto questo ho trovato sulla rete dei magnetotermici smart, che potrei sostituire ai miei. Il dubbio è che pochi ne parlano, ancora meno ne recensiscono e anche qui negli elenchi hardware non ci sono.

Accetto con umiltà ogni consiglio utile. Grazie!

**Comment by KC-inDomus** [comment]
Beh l'induzione se non hai presa dovresti avere differenziale sul quadro - quindi potresti usare il secondo canale del'EM per monitorarlo.

**Reply by mimelaine** [reply]
Si, infatti l'ho scritto... ma non posso spegnerla.
